### DISCUSSION TITLE
Meeting Notes 6/16

### DISCUSSION THREAD

**Discussion by voetberg** [deepskies/DeepBench#97] [General]
> In attendance - Becky, Omari, Brian, Maggie 

- Next week will be mostly joss paper 
- Maggie submitted the joss paper to export control
-

**Comment by bnord** [comment]
update contributors: 
1. @beckynevin 
2. @omaripaul57 

Everybody needs to update orcid

**Reply by bnord** [reply]
re-evaluate author order

**Reply by bnord** [reply]
@AeRabelais Do you want to change your last name in the Joss Paper?

**Reply by AeRabelais** [reply]
@bnord Yes!

**Comment by bnord** [comment]
Let's make a credit list thingy
CREDIT list: https://beta.elsevier.com/researcher/author/policies-and-guidelines/credit-author-statement?trial=true

**Reply by beckynevin** [reply]
I'm adding this to the acknowlegements section, feel free to add/erase anything I've already written :)

**Comment by bnord** [comment]
# Summary
1. add physics objects
2. first principles : mechanistic model; 
3. maybe explain a little more first principles and mechanisticness is important
4. add a clause about stochasticity and noise
5.

**Reply by beckynevin** [reply]
I've added these comments directly to the section in the overleaf.

**Comment by bnord** [comment]
# Statement of Need
1. not just talking about computer vision any more
2. not just talking about astronomy
3. ---> scope: physics based problems with statistical and learning-based approaches, -- e.g., uq crisis in deep learning
4. sort of like CIFAR10 but for physics-y-data

**Reply by bnord** [reply]
Let's re-think the feature list to have orthogonality of conceptualization

**Reply by bnord** [reply]
maybe second paragraph focuses on the need for data sets for new-fangled modeling techniques.

**Reply by bnord** [reply]
maybe expand beyond SBI

**Reply by beckynevin** [reply]
I started working on this. Feel free to add to/write over everything I've written.

**Comment by bnord** [comment]
https://www.overleaf.com/project/62853e0741343bad7edd6d6b
